commands/new_function_test.go

```diff
@@ -359,3 +359,36 @@ func tearDownNewFunction(t *testing.T, functionName string) {
 	}
 	handlerDir = ""
 }
+
+func Test_getPrefixValue_Default(t *testing.T) {
+	os.Setenv("OPENFAAS_PREFIX", "")
+	imagePrefix = ""
+
+	val := getPrefixValue()
+	want := ""
+	if val != want {
+		t.Errorf("want %s, got %s", want, val)
+	}
+}
+
+func Test_getPrefixValue_Env(t *testing.T) {
+	want := "alexellis"
+	os.Setenv("OPENFAAS_PREFIX", want)
+	imagePrefix = ""
+
+	val := getPrefixValue()
+	if val != want {
+		t.Errorf("want %s, got %s", want, val)
+	}
+}
+
+func Test_getPrefixValue_Flag(t *testing.T) {
+	want := "other"
+	os.Setenv("OPENFAAS_PREFIX", "")
```

Comment:
Not sure if this is the intent but in general had some problems with `os.Setenv` to `""`. Is it worth using `os.Unsetenv` to showcase empty variable? WDYT

Reply:
To unset environmental variable ^

Reply:
I'll try that